CityOfDetroit/bloom
# feat: new what to expect default text based on marketing

## Issue

- Closes #1293 

## Description

Adds new what to expect text based on the listing's marketing type, and removes the ability to edit those fields from the listing form.

## How Can This Be Tested/Reviewed?

Reseed on dev, then run the migration on this branch, and ensure the defaults are properly set on the existing listings and the fields are now gone in partners.

Additionally, ensure that when you create a new listing, the right text is set on both marketing types, and changes when the marketing type changes.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
